src/fswAlgorithms/sensorInterfaces/scanningInstrumentController/_UnitTest/test_scanningInstrumentController.py

```diff
@@ -0,0 +1,152 @@
+# 
+#  ISC License
+# 
+#  Copyright (c) 2023, Autonomous Vehicle Systems Lab, University of Colorado Boulder
+# 
+#  Permission to use, copy, modify, and/or distribute this software for any
+#  purpose with or without fee is hereby granted, provided that the above
+#  copyright notice and this permission notice appear in all copies.
+# 
+#  THE SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES
+#  WITH REGARD TO THIS SOFTWARE INCLUDING ALL IMPLIED WARRANTIES OF
+#  MERCHANTABILITY AND FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR
+#  ANY SPECIAL, DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES
+#  WHATSOEVER RESULTING FROM LOSS OF USE, DATA OR PROFITS, WHETHER IN AN
+#  ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF
+#  OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
+# 
+# 
+
+#
+#   Unit Test Script
```

Comment:
:large_blue_circle: This header information is redundant as it's captured in the Git history. Your call 😄

Reply:
It makes sense. I removed the header information.